bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -307,6 +318,7 @@
 		</supported-bridge-type-refs>
 		<label>@text/accessLabel</label>
 		<description>@text/accessDescription</description>
+		<semantic-equipment-tag>ControlDevice</semantic-equipment-tag>
```

Comment:
Should be the same as next one: Doorbell

Reply:
Ok.